crowdsec-docs/unversioned/cti_api/taxonomy/cti_fields.md

```diff
@@ -171,15 +174,75 @@ sidebar_position: 2
 The requested IP.
 
 ## `ip_range`
+
 > type: **string**
 
 ```json
-"ip_range": "1.2.3.0/24"
+"ip_range": "1.2.3.0/18"
 ```
 
 The range to which the IP belongs.
 
+## `reputation`
+
+> type: **string**
+
+```json
+"reputation" : "malicious"
+```
+
+The reputation of the IP address.
+
+The possible values are:
+
+- `malicious` : IP is malicious
+- `suspicious` : IP has been reported by many CrowdSec users but is not enough aggressive to be considered as malicious
+- `known` : IP is known in the CrowdSec network, but we don't have enough sightings
+- `safe` : IP is safe and can be trusted (ex: Google DNS, Cloudflare DNS ...)
+- `unknown`: IP is unknown or its last report is more than 3 month ago
+
+## `ip_range_24`
+
+> type: **string**
+
+```json
+"ip_range_24" : "1.2.3.0/24"
+```
+
+The /24 range of the IP address.
+
+## `ip_range_24_reputation`
+
+> type: **string**
+
+```json
+"ip_range_24_reputation" : "malicious"
+```
+
+The reputation of the /24 range of the IP address.
+
+The possible values are:
+
+- `malicious` : The range 24 of the IP is malicious
+- `suspicious` : The range 24 of the IP contains some IPs that has been reported by the CrowdSec network
+- `known` : The range 24 of the IP is known in CrowdSec network, but we don't have enough sightings about the its IP addresses
```

Comment:
The IP range is recognized in the CrowdSec network, but we lack sufficient sightings of its IP addresses.